sbin/prepareWorkspace.sh

```diff
@@ -224,9 +224,14 @@ checkingAndDownloadingFreeType()
       pngArg="--with-png=no";
     fi
 
+    local freetypeCC="";
+    if [[ "${BUILD_CONFIG[OS_ARCHITECTURE]}" == "i686" ]] || [[ "${BUILD_CONFIG[OS_ARCHITECTURE]}" == "i386" ]] ; then
+        freetypeCC="gcc -m32";
+    fi
+
     # We get the files we need at $WORKING_DIR/installedfreetype
     # shellcheck disable=SC2046
-    if ! (bash ./configure --prefix="${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}"/installedfreetype "${pngArg}" "${BUILD_CONFIG[FREETYPE_FONT_BUILD_TYPE_PARAM]}" && ${BUILD_CONFIG[MAKE_COMMAND_NAME]} all && ${BUILD_CONFIG[MAKE_COMMAND_NAME]} install); then
+    if ! (CC="${freetypeCC}" bash ./configure --prefix="${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}"/installedfreetype "${pngArg}" "${BUILD_CONFIG[FREETYPE_FONT_BUILD_TYPE_PARAM]}" && ${BUILD_CONFIG[MAKE_COMMAND_NAME]} all && ${BUILD_CONFIG[MAKE_COMMAND_NAME]} install); then
```

Comment:
If it's x64 then `CC=""` - That potentially overrides the default no?

Reply:
I've checked this on Debian amd64. `CC=""` behaves as expected and produces a 64 bit build of Freetype. It looks like `CC=""` is the same like not using this environment variable at all.

Reply:
Still makes me a little nervous :-).  Can you rework the logic so that the CC is only assigned in the case where it is x32?

Reply:
Okay. :)

Reply:
I've updated the pull request. Now the `CC` environment variable is only passed on i686 systems.

After Freetype is compiled the `CC` environment variable is **not** defined in the following `else` block and therefore won't interfere with following commands.

I've tested this on Debian amd64 - with a native 64bit build and with a 32bit cross compilation. In both cases Freetype is compiled for the correct architecture. Also a native build on Debian i386 works as expected.

I did not find a better solution than using `eval` without modifying the code too much. This command is also supported on macOS, which uses a pretty old version of Bash.

Reply:
great thank you!